src/pages/student/quizzes/QuizResult/index.js

```diff
@@ -90,12 +86,26 @@ const QuizResult = ({ score, total, quizId, categoryId }) => {
                           </div>
                           <tr>
                             <td>
-                              <a href="/#">
-                                <img
-                                  className={style.ResultsizeOfAvatar}
-                                  alt="avatar"
-                                  src={faker.image.avatar()}
-                                />
+                              <a>
+                                <span
+                                  onClick={() =>
+                                    redirectToStudDetail(friendScore.id)
+                                  }
+                                >
```

Comment:
Can we use <Link> here to replace the <a>? since we are using React Router.
Example of usage is here.
`src/pages/student/categories/CategoryList/index.js`

```suggestion
 <Link to={`/categories/${category.id}/sub`}>
               // jsx code here
 </Link>
```

or maybe you have another reason for using this?